### ISSUE TITLE
Sources for 3-beta7 ?

### ISSUE THREAD

**Issue by ChrisTitos** [andretietz/retroauth#83]
Hey (me again ;) )

I'm back to work on migrating to version 3, but I have an weird behavioral error (most likely related to the locking in the CredentialInterceptor) that I want to debug. But it seems like Android Studio cannot find the sources for beta7. So now I have no idea what's happening.

Are the sources included in the release? If not, could you do that? Thanks

**Comment by andretietz** [comment]
The source for the beta 7 are in the current master.

kotlin for now does not ship the sources with the common publishing tools. If you find a way, let me know.

**Comment by ChrisTitos** [comment]
I managed to compile the library locally and by changing [this line](https://github.com/andretietz/retroauth/blob/master/gradle/publish.gradle#L113) `sourceFiles `to `srcDirs` I was able to generate a source jar that wasn't empty and that I could then attach in my own project. (javadoc jar is still empty though). I have no experience with gradle publishing so I'm not sure if it is a viable fix or not, you decide.
